README.md

```diff
@@ -312,61 +315,61 @@ class MyClass {
 }
 ```
 
-Conditional statements are always required to be enclosed with braces,
-irrespective of the number of lines required.
+Conditional statements are always required to be enclosed with braces, irrespective of the number of lines required.
 
 __BAD:__
 
-```java
+```kotlin
 if (someTest)
-  doSomething();
-if (someTest) doSomethingElse();
+  doSomething()
+if (someTest) doSomethingElse()
 ```
 
 __GOOD:__
 
-```java
+```kotlin
 if (someTest) {
-  doSomething();
+  doSomething()
 }
-if (someTest) { doSomethingElse(); }
+if (someTest) { doSomethingElse() }
 ```
 
 
 ## Switch Statements
```

Comment:
Suggested change from Switch to When

## When Statements

Unlike switch statements in Java, when statements do not fall through. Separate cases by commas if they should be handled the same way.

Always include the `else` case.

__BAD:__

```kotlin
when (anInput) {
  1 -> doSomethingForCaseOne()
  2 -> doSomethingForCaseOneOrTwo()
  3 -> doSomethingForCaseThree()
}
```

__GOOD:__

```kotlin
when (anInput) {
  1, 2 -> doSomethingForCaseOneOrTwo()
  3 -> doSomethingForCaseThree()
  else -> println("No case satisfied")
}
```